src/routes/docs/products/auth/mfa/+page.markdoc

```diff
@@ -0,0 +1,1018 @@
+val response = account.createPhoneVerification()
+```
+```client-android-java
+account.createPhoneVerification(new CoroutineCallback<>((result, error) -> {
+   if (error != null)
+        error.printStackTrace();
+        return;
+    }
+
+    Log.d("Appwrite", result.toString());
+}));
+```
+{% /multicode %}
+
+After the user receives the verification code, they can verify their phone number by calling `account.updatePhoneVerification()`.
+{% multicode %}
+```client-web
+
+
+const response = await account.updatePhoneVerification(
+        '<USER_ID>',  // userId
+        '<SECRET>' // secret
+    );
+```
+```client-flutter
+Future result = account.updatePhoneVerification(
+    userId: '<USER_ID>',
+    secret: '<SECRET>',
+);
+
+result.then((response) {
+        print(response);
+    }).catchError((error) {
+        print(error.response);
+    });
+```
+```client-apple
+let token = try await account.updatePhoneVerification(
+  userId: "<USER_ID>",
+  secret: "<SECRET>"
+)
+```
+```client-android-kotlin
+val response = account.updatePhoneVerification(
+    userId = "<USER_ID>",
+    secret = "<SECRET>"
+)
+```
+```client-android-java
+account.updatePhoneVerification(
+    "<USER_ID>", // userId
+
+    "<SECRET>" // secret
+
+    new CoroutineCallback<>((result, error) -> {
+        if (error != null) {
+            error.printStackTrace();
+            return;
+        }
+
+        Log.d("Appwrite", result.toString());
+    })
+);
+```
+{% /multicode %}
+
+{% /tabsitem %}
+
+{% tabsitem #authenticator title="Authenticator" %}
+First, add a TOTP authenticator to the user's account by calling `account.addAuthenticator()`.
+
+{% multicode %}
+```client-web
+const { secret, uri } = await account.createMfaAuthenticator(
+        'totp' // type
+    );
+```
+```client-flutter
+Future result = account.createMfaAuthenticator(
+    type: 'totp',
+);
+
+result.then((response) {
+    print(response.secret);
+    print(response.uri);
+}).catchError((error) {
+    print(error.response);
+});
+```
+```client-apple
+let mfaType = try await account.createMfaAuthenticator(
+  type: "totp"
+)
+print(mfaType.secret)
+print(mfaType.uri)
+```
+```client-android-kotlin
+val response = account.createMfaAuthenticator(
+    type = "totp"
+)
+println(response.secret)
+println(response.uri)
+```
+```client-android-java
+account.createMfaAuthenticator(
+    "totp" // type
+
+    new CoroutineCallback<>((result, error) -> {
+        if (error != null) {
+            error.printStackTrace();
+            return;
+        }
+
+        Log.d("Appwrite", result.toString());
+    })
+);
+```
+{% /multicode %}
+
+This will create a secret and a URI. 
+The URI is a URL that can be used to generate a QR code for the user to scan with their TOTP authenticator app.
+
+You can generate a QR code for the user to scan by calling `avatars.getQR()`.
+
+{% multicode %}
+```client-web
+const result = await avatars.getQR(
+        authenticatorUrl,  // text
+        1,  // size (optional)
```

Comment:
Are we sure about size 1? Isn't that 1x1px? Wouldn't 0 be better? Lets just double check, no idea how API behaves in this scenario

Reply:
I did 800 instead. It defaults to 400, I tested around, 800 is a reasonable default.